stdlib/2/textwrap.pyi

```diff
@@ -1,29 +1,57 @@
-from typing import Any, AnyStr
+from typing import AnyStr, List
 
-class _unicode: ...
+class TextWrapper(object):
+    width: int = ...
+    expand_tabs: bool = ...
+    replace_whitespace: bool = ...
+    drop_whitespace: bool = ...
+    initial_indent: str = ...
+    subsequent_indent: str = ...
+    fix_sentence_endings: bool = ...
+    break_long_words: bool = ...
+    break_on_hyphens: bool = ...
 
-class TextWrapper:
-    whitespace_trans = ...  # type: Any
```

Comment:
This one does exist in Python 2.

Reply:
Oh whoops, my bad. I removed it mainly because it wasn't mentioned anywhere in the documentation.

I also took a look at the source code and saw that the TextWrapper class included a few extra attributes beyond `whitespace_trans` (e.g. `sentence_end_re` and `unicode_whitespace_trans`). I added those mainly for consistency -- should I leave those out?

Reply:
I don't feel strongly about it; let's just leave them in.